features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/postgresql.sql

```diff
@@ -788,3 +788,13 @@ CREATE TABLE IDN_SAML2_ASSERTION_STORE (
   PRIMARY KEY (ID)
 );
 
+DROP TABLE IF EXISTS IDN_OIDC_JTI;
+CREATE TABLE IDN_OIDC_JTI (
+  JWT_ID VARCHAR(255),
+  EXP_TIME TIMESTAMP DEFAULT 0,
+  TIME_CREATED TIMESTAMP DEFAULT 0,
+  PRIMARY KEY (JWT_ID)
+);
+
+CREATE INDEX IDX_AT_JI ON IDN_OIDC_JTI(JWT_ID);
```

Comment:
Since `JWT_ID` is a PK, do we have to explicitly add it as an index ?

Reply:
It will automatically create an unique index when an unique constraint or primary key is defined for a table. Hence removing the index.